### DISCUSSION TITLE
I am currently upgrading from halo1.6 to halo2.0, and I have only done a full site backup. The generated file is. zip, but the data. json file is required for migration. Is there a way to generate a. json file based on the full site backup file?

### DISCUSSION THREAD

**Discussion by coderhn** [halo-dev/halo#4540] [General]
I am currently upgrading from halo1.6 to halo2.0, and I have only done a full site backup. The generated file is. zip, but the data. json file is required for migration. Is there a way to generate a. json file based on the full site backup file?

**Comment by JohnNiang** [comment]
In Halo 2.x, we only support backing whole site backup, which format is ZIP file instead of JSON file in Halo 1.x.

If you want to restore the backup, you can request `/console/backup?tab=restore` directly and uploading your backup file.
